File: API/API/Model/Setting.cs
using System;
using System.Collections.Generic;

namespace API.Model;

public partial class Setting
{
    public uint SettingId { get; set; }

    public DateTime LastConnection { get; set; }

    public ulong IsOnline { get; set; }

    public ulong IsAdmin { get; set; }

    public ulong IsPrivate { get; set; }

    public ulong IsInfluencer { get; set; }

    public DateTime LastPasswordChanged { get; set; }

    public string LastPasswordHash { get; set; } = null!;

    public virtual ICollection<User> Users { get; set; } = new List<User>();
}

File: shopmate_app/app/src/main/java/com/example/shopmate_app/model/api/Setting.kt
package com.example.shopmate_app.model.api

data class Setting(
    val settingId: UInt?,
    val lastConnection: String,
    val isOnline: ULong,
    val isAdmin: ULong,
    val isPrivate: ULong,
    val isInfluencer: ULong,
    val lastPasswordChanged: String,
    val lastPasswordHash: String
)

data class SettingId(
    val settingId: String
)

class Settings : ArrayList<Setting>()